coreneuron/io/nrn_setup.cpp

```diff
@@ -394,6 +420,7 @@ void nrn_setup_cleanup() {
             delete[] netcon_srcgid[ith];
     }
     netcon_srcgid.clear();
+    netcon_negsrcgid_tid.clear();
```

Comment:
not sure if this has an impact on its usage in `netpar.cpp` ?

Reply:
```netcon_srcgid``` and ```netcon_negsrcgid_tid``` should stay together. Of course, that is not a direct answer to your question as maybe ```netcon_srcgid.clear()``` is not in its correct place either. In this case, I think they are both ultimately in the wrong place in a future refactoring where structure and data are separated to allow greater interactivity between NEURON and CoreNEURON. At present, however, the netpar.cpp use of these is in ```set_mindelay``` which is called
from within ```io/nrn_setup.cpp```.  As ```netcon_srcgid``` is in the class of largest arrays, it would be a good idea to review whether its existence is really necessary. But that is beyond the scope of this bugfix.